errdefs/helpers_test.go

```diff
@@ -8,6 +8,10 @@ import (
 
 var errTest = errors.New("this is a test")
 
+type wrapped interface {
```

Comment:
Looks like this is shadowed in various places in test, e.g. https://github.com/moby/moby/blob/453c165be709d294ab744f2efbd2552b338bb1a0/errdefs/helpers_test.go#L257-L260

So we should either rename the interface, or variables that are shadowing it.